src/openforms/js/components/admin/form_design/AuthPluginAutoLoginField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import Field from '../forms/Field';
+import {Checkbox} from '../forms/Inputs';
+
+const AuthPluginAutoLoginField = ({availablePlugins, selectedPlugins, selectedPlugin, onChange, errors}) => {
+    let filteredPlugins = availablePlugins.filter(plugin => selectedPlugins.includes(plugin.id))
+    const authCheckboxes = filteredPlugins.map(plugin => {
+        return (
+            <li key={plugin.id}>
+                <Checkbox
+                    name={`autoLoginAuthenticationBackend.${plugin.id}`}
+                    value={plugin.id}
+                    label={plugin.label}
+                    onChange={onChange}
+                    checked={selectedPlugin === plugin.id}
+                />
+            </li>
+        );
+    });
+
+    return (
+        <Field
+            name="formAuthPluginAutoLogin"
```

Comment:
the `Field` component forwards the `name` prop to its children, see https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/admin/forms/Field.js#L29

However, this doesn't work properly here because of the nested `<ul>`. Instead, you should let `AuthPluginAutoLoginField` take the `name` prop and make that the direct child of the `Field` component, and that component should in turn render the `<ul>`.

The correct approach is:

```jsx
<Field name="foo" {...rest}>
    <AuthPluginAutoLoginFieldOptions {...relevantProps} />
</Field>
```

with 

```jsx
const AuthPluginAutoLoginFieldOptions = ({name, ...relevantProps}) => (
    <ul>
        {plugins.map(plugin => (
            <li>
                <Radio name={name} {...radioProps} />
            </li>
        ))}
    </ul>
);
```

or keep them together in a single component but then repeat/fix the `name` prop.